docs/guidance/ingress.md

```diff
@@ -39,6 +39,7 @@ eksctl get cluster ${YOUR_EKS_CLUSTER} # Check subnets on the EKS cluster
 
 # Step 4: Create an ALB ingress. When an ingress with proper annotations creates,
 #        AWS Load Balancer controller will reconcile a ALB (not in AWS EKS cluster).
+# For RayService, you can use ray-operator/config/samples/ray_v1alpha1_rayservice-alb-ingress.yaml
```

Comment:
Maybe we should have a separate section for RayService.

Reply:
we will have it in the ray doc.

Reply:
Based on our previous discussion, it appears that we have decided to place the "Kubernetes-specific" documentation in the KubeRay repository and the "Ray-specific" documentation in the Ray repository.